app/models/service.rb

```diff
@@ -65,6 +65,7 @@ class Service < ApplicationRecord
   include TenancyMixin
   include SupportsFeatureMixin
   include Metric::CiMixin
+  include ResourceLinking
```

Comment:
Shall we name it `ResourceLinkingMixin` similar to lines above?

Reply:
I was suggesting we use `include_concern` here.